sdk/communication/AzureCommunicationCalling/CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Release History
 
+## 2.0.0-beta.1 (2021-07-14)
+### New features
+- `Recording` and `Transcription` features are decoupled from `Call` object and now can only be used via extensions.
+  For e.g. 
+```
+            let recordingFeature = self.call!.api(RecordingFeature.self)
+            recordingFeature.delegate = self.callObserver
+```
+
+### Breaking changes
+- `didChangeRecordingState` and `didChangeTranscriptionState` are moved out of `CallDelegate` into `RecordingFeatureDelegate` and into `TranscriptionFeatureDelegate`.
+
+**More documentation on extensions and the breaking change can be found [here](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/calling-client-samples?pivots=platform-ios).**
+
+*NOTE: Version previously released `1.2.0-beta.1` and this version only has the difference in version number and this version will be available in cocoapods.*
```

Comment:
This note is a little confusing. I'd recommend:
"NOTE: Previously released version `1.2.0-beta.1` is functionally identical to `2.0.0-beta.1`."

Reply:
This was moved but still confusing.